### PR TITLE
Apply is_valid_file validation from v2 validation to every version number

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/38487

## Description
Validation of correct display name for every versioned script or integration to be supported by any v<number> and not only v2

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -1554,20 +1554,24 @@ def arg_to_list(arg: Union[str, List[str]], separator: str = ",") -> List[str]:
     return [arg]
 
 
-def is_v2_file(current_file, check_in_display=False):
```
Comment (roysagi) [inline demisto_sdk/commands/common/tools.py:1557]: please make sure this method is not used anywhere else.

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 2]
```diff
@@ -954,13 +954,15 @@ def all_feed_params_exist(self) -> bool:
 
     def is_valid_display_name(self):
         # type: () -> bool
-        if not is_v2_file(self.current_file, check_in_display=True):
+        maybe_version_number: Optional[str] = get_file_version_suffix_if_exists(self.current_file,
```
Comment (roysagi) [inline demisto_sdk/commands/common/hook_validations/integration.py:957]:
```suggestion
        version_number: Optional[str] = get_file_version_suffix_if_exists(self.current_file,
```

File: demisto_sdk/commands/common/tools.py (hunk 1 above)
Comment (tomneeman151293) [reply demisto_sdk/commands/common/tools.py:1557]: Made sure

**Review by dantavori — APPROVED** [review]
awesome

**Pull request merged**